PlayerInfo/DeviceSettings/PlatformImplementation.cpp

```diff
@@ -520,7 +520,7 @@ class PlayerInfoImplementation : public Exchange::IPlayerProperties, public Exch
         {"2160p25", RESOLUTION_2160P25},
         {"2160p50", RESOLUTION_2160P50},
         {"2160p30", RESOLUTION_2160P30},
-        {"2160p60", RESOLUTION_2160P60},
```

Comment:
There is already entry for 2160p. Checking why are we changing this entry for 2160p60